unit_test/blas/Test_Blas_gesv.hpp

```diff
@@ -139,8 +157,26 @@ void impl_test_gesv_mrhs(const char* mode, const char* padding, int N, int nrhs)
     }
 
     // Solve.
-    KokkosBlas::gesv(A,B,ipiv);
-    Kokkos::fence();
+    try {
+      KokkosBlas::gesv(A,B,ipiv);
+      Kokkos::fence();
```

Comment:
Note: the fence will on execute if `KokkosBlas::gesv` does not throw a runtime_error.

Reply:
I will move fence out of ```try```. Thanks for pointing it out.